crashreporter/src/main/java/com/tealium/crashreporter/CrashReporter.kt

```diff
@@ -0,0 +1,41 @@
+package com.tealium.crashreporter
+
+import com.tealium.crashreporter.CrashReporter.Tracker
+import com.tealium.internal.data.Dispatch
+import com.tealium.internal.listeners.ActivityResumeListener
+import com.tealium.internal.listeners.DisableListener
+import com.tealium.internal.listeners.PopulateDispatchListener
+import com.tealium.library.DataSources
+import com.tealium.library.Tealium
+
+/**
+ * Crash Reporter module for tracking crash data. When used, this module populates
+ * <i>crash_*</i> attributes defined in {@link com.tealium.library.DataSources.Key}.
```

Comment:
This is in Javadoc format; kotlin's doc style and the way they create links to other classes is different.
Also, `com.tealium.library.DataSources` doesn't exist as a class in the new Kotlin lib.